recipes/libsystemd/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.29.0":
+    url: "https://github.com/systemd/systemd-stable/archive/v246.6.tar.gz"
```

Comment:
I am curious... why these are different?

Reply:
Hm. Looks like i misunderstood about libsystemd versioning.
0.29.0 is a libsystemd.so version.
246.6 is a systemd source package version.

libsystemd.pc file contains systemd version. So the version should be 246.6 in both places.